packages/react-image/package.json

```diff
@@ -41,8 +41,6 @@
   "dependencies": {
     "@fluentui/react-make-styles": "^9.0.0-alpha.24",
     "@fluentui/react-utilities": "^9.0.0-alpha.17",
-    "@microsoft/load-themed-styles": "^1.10.26",
```

Comment:
We don't need this dependency there.

Reply:
I see all the `object-fit` styles commented out currently, then yeah in that case let's remove it for now